types/babel-plugin-macros/index.d.ts

```diff
@@ -23,6 +23,7 @@ declare namespace babelPluginMacros {
         references: { default: Babel.NodePath[] } & References;
         state: Babel.PluginPass;
         babel: typeof Babel;
+        config?: { [key: string]: any };
```

Comment:
OPTIONAL: It would be nice if this was a generic which would allow specifying the shape of the expected configuration.